gui/wxpython/datacatalog/infomanager.py

```diff
@@ -0,0 +1,31 @@
+
+import wx
+import webbrowser
+
+from grass.script import gisenv
+
+
+class InfoManagerDataCatalog:
+    """
+    InfoBar Manager for Data Catalog
+    """
+    def __init__(self, infobar, sizer):
+        self.infoBar = infobar
+        self.sizer = sizer
+
+    def ShowInfoBar1(self, buttons):
```

Comment:
Let's describe the purpose and goal in name and docstring:

```suggestion
    def ShowDataStructureInfo(self, buttons):
    """Show info about the data hierarchy focused on the first-time user"""
```